fusillade/clouddirectory.py

```diff
@@ -1410,10 +1410,15 @@ def __init__(self, name: str = None, object_ref: str = None):
 
     def lookup_policies(self) -> List[str]:
         try:
-            policy_paths = self.lookup_policies_batched()
-        except cd_client.exceptions.ResourceNotFoundException:
+            status = self.status
```

Comment:
This property is expected to occasionally raise an error upon access? This behavior seems confusing. Consider creating a utility method to provide this check.

Reply:
added `User.is_enabled()` function